Kickstarter-iOS/Views/Controllers/LoginToutViewController.swift

```diff
@@ -440,7 +486,20 @@ private let baseStackViewStyle: StackViewStyle = { stackView in
 private let baseLabelStyle: LabelStyle = { label in
   label
     |> \.textAlignment .~ NSTextAlignment.center
-    |> \.backgroundColor .~ UIColor.white
     |> \.lineBreakMode .~ NSLineBreakMode.byWordWrapping
     |> \.numberOfLines .~ 0
 }
+
+private let logoImageViewStyle: ImageViewStyle = { imageView in
+  imageView
+    |> \.image .~ image(named: "kickstarter-logo")?.withRenderingMode(.alwaysTemplate)
+    |> \.tintColor .~ .ksr_green_500
+    |> \.contentMode .~ .scaleAspectFit
+    |> \.translatesAutoresizingMaskIntoConstraints .~ false
+}
+
+private let separatorViewStyle: ViewStyle = { view in
```

Comment:
We already have `separatorStyle` in `BaseStyles.swift`. We should try to avoid having many different styles for the same component, would the one we already have work?